website/docs/reference/developer-readme.md

```diff
@@ -6,22 +6,35 @@ description: SynapseML Development Setup
 ---
 
 # SynapseML Development Setup
-
+1) Install JDK 11
+    - https://docs.microsoft.com/en-us/java/openjdk/download#openjdk-11
+1) On Windows, Configure Hadoop "winutils"
```

Comment:
I think this guide would be MUCH better if we had separate sections for Windows, Linux and MacOS.  Trying to follow these steps and keeping in mind what is windows vs linux vs macos is going to be hard.  Also, we should add some HTML to collapse the sections.  Take a look at the lightgbm guide for reference, it is much better than this:
https://github.com/microsoft/LightGBM/blob/master/docs/Installation-Guide.rst
https://lightgbm.readthedocs.io/en/latest/Installation-Guide.html

Especially the html - it has collapsed sections for different types of installations users may want to do.

Reply:
Also, instead of putting this in developer-readme.md, it would be good if we just had a separate installations file.  But it might be out of scope for this PR.

Reply:
draft of linux steps stubbedout.